Samples/characterization/phase-estimation/Program.qs

```diff
@@ -0,0 +1,30 @@
+﻿namespace PhaseEstimationSample {
```

Comment:
Similarly here, I'd suggest a copyright header and using the same namespace.

Reply:
done